dlrover/python/training_event/predefined/_dlrover.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2025 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+from typing import List, Optional
+
+from dlrover.python.common.singleton import Singleton
+from dlrover.python.training_event.predefined.common import CommonPredefined
+
+
+class DLRoverCommonEventName(Enum):
+    """Event in DLRover agent and master"""
+
+    DLROVER_START = "#start"
+    DLROVER_NODE_JOIN = "#node_join"
+    DLROVER_RENDEZVOUS = "#rendezvous"
+    DLROVER_NETWORK_CHECK = "#network_check"
+    DLROVER_ELASTIC_TRAIN = "#elastic_train"
+    DLROVER_PROCESS_RESTART = "#process_restart"
+    DLROVER_EXIT = "#exit"
+
+
+class DLRoverMasterEventName(Enum):
+    """Event in DLRover master"""
+
+    DLROVER_POD_CREATE = "#pod_create"
+    DLROVER_POD_CHANGE = "#pod_change"
+    DLROVER_POD_RELAUNCH = "#pod_relaunch"
+    DLROVER_FAULT_DETECT = "#fault_detect"
+    DLROVER_REPAIR = "#repair"
+
+
+class DLRoverCommon(CommonPredefined):
```

Comment:
The naming in this section needs to be consistently revised (including in the documentation). This naming may cause ambiguity with DLRover's own concept of an Agent. It would be best to add an identifier like "event" as a prefix or suffix to distinguish and clarify.

e.g.
```DLRoverCommonEvent```,```DLRoverAgentEvent```